apps/server/src/form-instances/form-instances.service.ts

```diff
@@ -125,6 +135,14 @@ export class FormInstancesService {
                 signerType: 'USER',
                 assignedUserId: employeeId,
               },
+              {
```

Comment:
We look for any employee in the assignedUserList that matches the current employee's id